### ISSUE TITLE
Cannot get the reported results.

### ISSUE THREAD

**Issue by LiaoYuxuan** [yinboc/prototypical-network-pytorch#17]
According to the paper, using 5-way 5-shot to train and 5-way 5-shot to test, the accuracy should be ~65.77%, but I can only get ~43.79%. I don't know why, could you please give me any suggestions?

**Comment by Chris-G-X** [comment]
I can only get ~47.20%.。it has some problems in the code
